<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DLAP-0317" failures="1" tests="2" name="AK44" time="23.448" errors="0" timestamp="2024-10-11T03:50:19 ICT">
  <testcase classname="automation.demo.Listener.ListenerTestCases" name="TestFail" time="10.384">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at automation.demo.Listener.ListenerTestCases.TestFail(ListenerTestCases.java:57)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- TestFail -->
  <testcase classname="automation.demo.Listener.ListenerTestCases" name="TestPass" time="12.989"/>
</testsuite> <!-- AK44 -->
